lib/pages/chat/chat.dart

```diff
@@ -1304,6 +1317,114 @@ class ChatController extends State<Chat>
     );
   }
 
+  void onHover(bool isHovered, int index, Event event) {
```

Comment:
what if index is 0, and also is it the index of context menu item

Reply:
IF INDEX = 0 CHAT LIST SHOW EMPTY WIDGET
<img width="274" alt="Screenshot 2023-08-30 at 15 44 35" src="https://github.com/linagora/twake-on-matrix/assets/99852347/8b7d7ff8-7cd5-4c93-b62c-bcc66a69d578">
<img width="367" alt="Screenshot 2023-08-30 at 15 44 57" src="https://github.com/linagora/twake-on-matrix/assets/99852347/1fd9ff64-b2ce-4b7f-a47c-2d31b8f5aeb3">

Reply:
yeah, i know but you should also check a condition inside, because the function can be reused

Reply:
If you read it carefully, you know enough that it cannot be < 1.
1. I'll update check condition on inside func
  2. Not reuse, because it depends on `ValueNotifier ` at that exact screen
<img width="592" alt="Screenshot 2023-08-30 at 16 09 05" src="https://github.com/linagora/twake-on-matrix/assets/99852347/78b0b369-0aea-433c-ae42-1e5a08d94b7f">

Reply:
should check it, pleaseeee